CapitalizeDecorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative "BaseDecorator.rb"
+
+class CapitalizeDecorator < BaseDecorator
+    def correct_name
+        super.capitalize
+    end
```

Comment:
[REQUIRED] Your correct_name should look something like this:
```ruby
  def correct_name
      @nameable.correct_name.capitalize
    end
```

Reply:
Your nice feedback is really appreciated 🙏.

- [ ] I have checked, and made it fix 🎯. The changes applied to the `CapitalizeDecorator` class.